ci/build.py

```diff
@@ -305,61 +246,7 @@ def container_run(platform: str,
                     {'bind': '/work/ccache', 'mode': 'rw'},
             },
             environment=environment)
-        try:
-            logging.info("Started container: %s", trim_container_id(container.id))
-            # Race condition:
```

Comment:
So this is no longer an issue?

Reply:
It's still a thing - the signal blocking has moved to the "safe_docker_run.py" script ^^

Reply:
I'll move these comments in there for clarity though - good catch!